Application/Transfer/Import/Standard/Service.php

```diff
@@ -0,0 +1,1561 @@
+            '',
+            $firstName,
+            $secondName,
+            $lastName,
+            $GroupList,
+            '',
+            '',
+            $callName
+        );
+    }
+
+    /**
+     * @param string $salutation
+     * @param string $title
+     * @param string $firstName
+     * @param string $lastName
+     * @param string $memberNumber
+     * @param string $assistance
+     *
+     * @return bool|TblPerson
+     */
+    private function setPersonCustody($salutation, $title, $firstName, $lastName, $memberNumber, $assistance)
+    {
+
+        $GroupList = array();
+        $GroupList[] = Group::useService()->getGroupByMetaTable(TblGroup::META_TABLE_COMMON);
+        $GroupList[] = Group::useService()->getGroupByMetaTable(TblGroup::META_TABLE_CUSTODY);
+        if($memberNumber){
+            $GroupList[] = Group::useService()->getGroupByMetaTable(TblGroup::META_TABLE_CLUB);
+        }
+
+        $tblSalutation = false;
+        $gender = 0;
+        if($salutation){
+            $salutation = strtolower($salutation);
+            switch($salutation){
+                case 'herr':
+                case 'm':
+                case 'h':
+                    $tblSalutation = Person::useService()->getSalutationByName(TblSalutation::VALUE_MAN);
+                    $gender = TblCommonGender::VALUE_MALE;
+                break;
+                case 'frau':
+                case 'w':
+                case 'f':
+                    $tblSalutation = Person::useService()->getSalutationByName(TblSalutation::VALUE_WOMAN);
+                    $gender = TblCommonGender::VALUE_FEMALE;
+                break;
+            }
+        }
+
+        $tblPerson = Person::useService()->insertPerson(
+            $tblSalutation,
+            $title,
+            $firstName,
+            '',
+            $lastName,
+            $GroupList
+        );
+
+        $isAssistance = TblCommonInformation::VALUE_IS_ASSISTANCE_NULL;
+        if($assistance){
+            $isAssistance = TblCommonInformation::VALUE_IS_ASSISTANCE_YES;
+        }
+
+        Common::useService()->insertMeta(
+            $tblPerson,
+            '',
+            '',
+            $gender,
+            '',
+            '',
+            $isAssistance,
+            $assistance,
+            ''
+        );
+
+        Club::useService()->insertMeta($tblPerson, $memberNumber);
+
+        return $tblPerson;
+    }
+
+    /**
+     * @param TblPerson $tblPerson
+     * @param string    $birthdayString
+     * @param string    $birthPlace
+     * @param string    $gender
+     * @param string    $nationality
+     * @param string    $denomination
+     * @param string    $remark
+     * @param int       $RunY
+     * @param string    $Nr
+     * @param array     $error
+     */
+    private function setPersonBirth(TblPerson $tblPerson, $birthdayString, $birthPlace, $gender, $nationality, $denomination, $remark, $RunY, $Nr, &$error)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method setPersonBirth has 10 parameters. Consider reducing the number of parameters to less than 10.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)